lib/charms/loki_k8s/v0/loki_push_api.py

```diff
@@ -246,6 +247,30 @@ def __init__(self, *args):
 - `juju_model_uuid`
 - `juju_application`
 
+
+Whether alert rules files does not contain the keys `alert` or `expr` or
+there is no alert rules file in `alert_rules_path` a `loki_push_api_alert_rules_error` event
+is emitted.
+To handle these situations the event must be observed in the `LokiClientCharm` charm.py file:
+
+
+```python
+class LokiClientCharm(CharmBase):
+
+    def __init__(self, *args):
+        super().__init__(*args)
+        ...
+        self._loki_consumer = LokiPushApiConsumer(self)
```

Comment:
I wonder if, in line with google docstring style, LokiPushApiConsumer should have an `Emits:` section listing all the custom events.

Reply:
Sounds reasonable to me. 👍🏽

Reply:
Good catch @sed-i !